types/vinyl-fs/index.d.ts

```diff
@@ -20,128 +20,86 @@ import * as File from 'vinyl';
 import * as globStream from 'glob-stream';
 
 export interface SrcOptions extends globStream.Options {
-    /** Prevents stream from emitting an error when file not found. */
-    allowEmpty?: boolean | undefined;
-
-    /** Specifies the working directory the folder is relative to */
-    cwd?: string | undefined;
-
-    /**
-     * Specifies the folder relative to the cwd
-     * This is used to determine the file names when saving in .dest()
-     * Default: where the glob begins
-     */
-    base?: string | undefined;
```

Comment:
Those options are removed because they are actually glob-stream options, and so already covered by `extends globStream.Options`

Reply:
True, although the documentation is different and/or missing from these options in `glob-stream` and `glob` so these comments will be lost.